### PR TITLE
Ft legume

### PR DESCRIPTION
**Pull request by AndrewMuchiri**

### CODE REVIEW

File: customer.html [hunk 1]
```diff
@@ -70,6 +70,17 @@ <h3>LEGUMES</h3>
...
+                <div class="form-group">
+                  <label for="quantity">Enter the quantity: </label>
+                  <input type="number" name="quantity" class="form-control quantity" value="" required>
+                </div>
+                <input type="submit" class="submit" name="submit" value="order">
+                <button type="button" class="another-order" name="button">PlACE ANOTHER ORDER</button>
+              </div>
+              <div >
+                <ul class="charge1-customers">
+
```
Comment (Vincent-kipngeno) [inline customer.html:77]: charge1-customers could you change it to charge(no of your module)-customers

File: farmers.html [hunk 2]
```diff
@@ -50,3 +50,36 @@ <h3>ANIMAL PRODUCTS</h3>
...
+                 <div class="form-group">
+                   <label for="quantity">Enter the quantity: </label>
+                   <input type="number" name="quantity" class="form-control quantity" value="" required>
+                 </div>
+                 <input type="submit" class="submit" name="submit" value="order">
+                 <button type="button" class="another-order" name="button">PlACE ANOTHER ORDER</button>
+               </div>
+               <div >
+                 <ul class="charge1-customers">
+
```
Comment (Vincent-kipngeno) [inline farmers.html:77]: could you change charge1-customers to charge4-farmers

**Comment by NinahMo** [comment]
good job

**Comment by Vincent-kipngeno** [comment]
good work

**Pull request merged**